tests/Message/CreateOrderRequestTest.php

```diff
@@ -159,6 +159,37 @@ public function testGetLines()
         $this->assertSame('121.14', $line['vatAmount']['value']);
     }
 
+    public function testDiscountLines()
+    {
+        $this->request->setLines([
+            [
+                'type' => 'physical',
+                'sku' => '5702016116977',
+                'name' => 'LEGO 42083 Bugatti Chiron',
+                'quantity' => 2,
+                'vatRate' => '21.00',
+                'unitPrice' => '399.00',
+                'totalAmount' => '698.00',
+                'discountAmount' => '100.00',
+                'vatAmount' => '121.14',
+            ],
+            [
+                'type' => 'discount',
+                'name' => 'Discount 100 EURO',
+                'quantity' => 1,
+                'vatRate' => '21.00',
+                'unitPrice' => '-100.00',
+                'totalAmount' => '-100.00',
+                'vatAmount' => '-21.00',
```

Comment:
This is a bit tricky - but for this unit price the `vatAmount` must be `17,36`.

Reply:
Okay, shouldn't matter for the test but fixed in https://github.com/thephpleague/omnipay-mollie/commit/ee3604c29cb85ecc9be580558fdcacf8bc994575

Reply:
Right. But we don't want to give anyone the impression that unitPrice or totalAmount are without VAT.

Reply:
Thanks for updating @barryvdh